pkgs/by-name/le/legba/package.nix

```
@@ -0,0 +1,29 @@
+{ lib, rustPlatform, fetchFromGitHub, cmake, pkg-config, openssl }:
```

Easier to read diff in the future.

```suggestion
{ lib
, rustPlatform
, fetchFromGitHub
, cmake
, pkg-config
, openssl
}:
```